idd/Energy+.idd.in

```diff
@@ -818,6 +818,7 @@ HVACSystemRootFindingAlgorithm,
        \key Bisection
        \key BisectionThenRegulaFalsi
        \key RegulaFalsiThenBisection
+       \key Alternation
```

Comment:
Alternation.....sounds like an Alternative Rock Radio Station name: 104.3 The AlterNation!